app/views/api/umgap/casestudy_preconfigured.html.erb

```diff
@@ -0,0 +1,137 @@
+  <div class="col-xs-9">
+    <h1 class="api-title">Case Study: Running a preconfigured metagenomics pipeline</h1>
+    <p class="lead">This case study shows how to set up and run the preconfigured scripts of the <span class="initialism">UMGAP</span> for the taxonomic analysis of a metagenomics dataset.</p>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Introduction</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          The Unipept Metagenomics Analysis Pipeline (UMGAP) is accompanied by 3 scripts:
+          <code>umgap-setup.sh</code>, <code>umgap-analyse.sh</code> and <code>umgap-visualize.sh</code>.
+          The setup script deals with downloading prebuilt databases and checking and linking external tools.
+          The analyse script runs several analyses in sequence.
+          The visualize script creates webpages from the results of the analyse script.
+        </p>
+        <p>
+          The following pipelines are preconfigured: <ul>
+            <li><strong><span class='initialism'>UMGAP</span> High Precision:</strong> Optimized for high precision identifications on your metagenomics reads.</li>
+            <li><strong><span class='initialism'>UMGAP</span> Max Precision:</strong> Optimized for highest precision, with a small setback on sensitivity.</li>
+            <li><strong><span class='initialism'>UMGAP</span> Tryptic Precision:</strong> Made for fast analyses on your laptop. Fewer results, but accurate.</li>
+            <li><strong><span class='initialism'>UMGAP</span> High Sensitivity:</strong> Optimized for high sensitivity identifications on metagenomics reads.</li>
+            <li><strong><span class='initialism'>UMGAP</span> Max Sensitivity:</strong> Optimized for highest sensitivity, with a small setback on precision.</li>
+            <li><strong><span class='initialism'>UMGAP</span> Tryptic Sensitivity:</strong> Made for fast analyses on your laptop. Get a quick overview of your data.</li>
+          </ul>
+        </p>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Setup</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          The preconfigured pipelines require some databases and external tools.
+          The setup script will help you to get these in the right place.
+          If you're planning to use the tryptic precision, tryptic sensitivity, high precision or max precision pipelines,
+           you'll need to <a href="https://github.com/unipept/FragGeneScanPlusPlus/">install FragGeneScan++</a>.
+          In this casestudy, we will use the two tryptic pipelines, so we don't have to download the (large) 9-mer index.
+        </p>
+        <pre style="max-height: initial;">
+<!--   --><b>$</b> umgap-setup.sh -f /opt/FragGeneScanPlusPlus
+<!--   -->Use '/home/ninewise/.config/unipept' as configuration directory? [y]/n y
```

Comment:
replace `ninewise` with `user` or something like that so the reader is not confused